packages/aws-cdk-lib/aws-s3/test/bucket.test.ts

```diff
@@ -879,6 +879,28 @@ describe('bucket', () => {
     });
   });
 
+  test('bucket with default block public access setting to throw error msg', () => {
+    const stack = new cdk.Stack();
+
+    expect(() => new s3.Bucket(stack, 'Bucket', {
+      publicReadAccess: true,
+    })).toThrow('Cannot use \'publicReadAccess\' property');
```

Comment:
error msg needs updating?

```suggestion
    })).toThrow(''Cannot use \'publicReadAccess\' property on a bucket without allowing bucket-level public access through \'blockPublicAceess\' property.');
```